mon-pix/app/styles/pages/_send-profile.scss

```
@@ -31,10 +31,27 @@
     flex-flow: column;
     align-items: center;
     border-radius: 6px;
-    background-color: $grey-10;
+    background-color: $silver-grey;
     padding: 20px;
   }
 
+  &__profile {
+    border-top: 1px dashed $silver-grey;
+    border-bottom: 1px dashed $silver-grey;
+    padding-top: 36px;
+    padding-bottom: 22px;
+    margin-top: 24px;
+    margin-bottom: 39px;
+    display: flex;
+    flex-flow: column;
+    align-items: center;
+
+    &__cards {
+      margin-top: 24.75px;
```

```suggestion
      margin-top: 24px;
```
ou ```suggestion
      margin-top: 25px;
```
le .75 est un peu overkill je pense

---

Done